### PR TITLE
Match Path Payments from Clients

### PR DESCRIPTION
**Pull request by JakeUrban**
resolves stellar/django-polaris#355

The issue describes the problem better, but basically we need to support matching streamed payments in all forms, not just via payment operations. So I added support for matching path payment ops, both strict send and strict receive.

### CODE REVIEW

**Comment by JakeUrban** [comment]
Still updating the docs but the code is done

**Comment by JakeUrban** [comment]
Actually, I didn't need to change much on the docs because it is expected that path payments work, since path payments are the basis for SEP-31 cross-border payments.

File: polaris/polaris/management/commands/watch_transactions.py [hunk 1]
```diff
@@ -95,16 +107,13 @@ def process_response(cls, response, account):
             return
 
         try:
-            stellar_transaction_id = response["id"]
+            _ = response["id"]
```
Comment (Zagan202) [inline polaris/polaris/management/commands/watch_transactions.py:110]: Could this be removed if we're not using it?

**Review by Zagan202 — APPROVED** [review]
Besides 1 comment LGTM

**Pull request merged**